

try:
    print(m)
except Exception as msg:
    print(msg)

print(m)